CHANGELOG.md

```diff
@@ -1,25 +1,33 @@
 # Changelog
 
-## 7.17.0
-
-### Fixes
-
-- Fix transaction end timestamp trimming ([#1916](https://github.com/getsentry/sentry-dart/pull/1916))
-  - Transaction end timestamps are now correctly trimmed to the latest child span end timestamp
-- remove transitive dart:io reference for web ([#1898](https://github.com/getsentry/sentry-dart/pull/1898))  
-
-### Features
+## Unreleased
 
+- Add TTFD (time to full display), which allows you to measure the time it takes to render the full screen ([#1920](https://github.com/getsentry/sentry-dart/pull/1920))
+  - Requires using the [routing instrumentation](https://docs.sentry.io/platforms/flutter/integrations/routing-instrumentation/).
+  - Set `enableTimeToFullDisplayTracing = true` in your `SentryFlutterOptions` to enable TTFD
+  - Manually report the end of the full display by calling `SentryFlutter.reportFullyDisplayed()`
+  - If not reported within 30 seconds, the span will be automatically finish with the status `deadline_exceeded`
 - Add TTID (time to initial display), which allows you to measure the time it takes to render the first frame of your screen ([#1910](https://github.com/getsentry/sentry-dart/pull/1910))
```

Comment:
Is TTID not released, yet?

Reply:
craft messed up the merging. I merged the ttid pr during the release of 7.17.0 so craft merged the changelog together in a wrong way when the release finished - but it's not included in the release changelog so all good